src/lib/openapi/spec/context-field-strategies-schema.ts

```diff
@@ -1,7 +1,7 @@
 import { FromSchema } from 'json-schema-to-ts';
 
 export const contextFieldStrategiesSchema = {
-    $id: '#/components/schemas/segmentStrategiesSchema',
```

Comment:
the ID here looks like a bug to me, do you agree @sjaanus?

Reply:
Yes, completely agree.